modules/uid2IdSystem_shared.js

```diff
@@ -160,18 +467,124 @@ function refreshTokenAndStore(baseUrl, token, clientId, storageManager, _logInfo
       originalToken: token,
       latestToken: response.identity,
     };
+    let storedTokens = storageManager.getStoredValueWithFallback();
```

Comment:
What is this doing?

Reply:
Get back stored uid2 tokens from cookies or localStorage. I add this line here to keep tracking the originalIdentity